app/models/legal_aid_application.rb

```diff
@@ -129,8 +129,8 @@ def set_transaction_period
     date_from = date_to - 3.months
 
     update!(
-      transaction_period_start_on: date_from,
-      transaction_period_finish_on: date_to
+      transaction_period_start_on: date_from.beginning_of_day,
+      transaction_period_finish_on: date_to.beginning_of_day
     )
```

Comment:
This won't have any effect because the `transaction_period_start_on` and `transaction_period_finish_on` columns are defined as dates, not datetimes.

Reply:
I have reverted this change... we only need to store the date and send a datetime to true-layer
As the time element is always defaulted to 00:00:00 storing it
seems superfluous